### ISSUE TITLE
Whiteboard does not show up for other participants when opened by moderator and reports “Couldn't decrypt data.” when opened manually

### ISSUE THREAD

**Issue by ntninja** [jitsi/jitsi-meet#14001]
### Description:

Whiteboard does not show up for other participants when opened by moderator and displays are barrage of “Couldn't decrypt data.” alert() when opened manually by another participant. Would be thankful for any hints as to why this may occur.

### Steps to reproduce:

1. Install Jitsi Meet on NixOS
2. Configure Prosody in NixOS to also enable the room_metadata and pubsub extensions
3. Run jitsi-excalidraw-backend server behind HTTP/WSS reverse proxy
4. Add whiteboard configuration to `var config` (whiteboard.enabled: true & whiteboard.collabServerBaseUrl)
5. Start a new meeting in Jitsi Meet
6. Open the whiteboard
7. Verify that whiteboard successfully can establish a WebSocket connection to the configured jitsi-excalidraw-backend server (it does)

### Expected behavior:

Whiteboard should show up for other meeting participants.

### Actual behavior:
Whiteboard does not show up for other meeting participants. If another participant opens the whiteboard manually as well, this will spam the other participant with “Couldn't decrypt data.” alert() messages with another barrage of messages displayed each time they move the mouse cursor or try to draw anything. Each user will only see their local whiteboard changes, but never anything drawn by others.

### Server information:

- Jitsi Meet version: 1.0.6943
- Operating System: NixOS 23.05

### Client information:

- Browser / app version: Brave latest, Chromium latest, Firefox DevEdition latest (any combination)
- Operating System: Debian unstable

### Additional information:

The Jitsi Meet server uses the Secure Domain feature with Prosody internal user authentication.

EDIT: Fixed Jitsi Meet version number.

**Comment by ntninja** [comment]
I think this is the same issue as https://github.com/jitsi/excalidraw-backend/issues/9, but I don’t think that the issue is in the backend since the WebSocket connects just fine.

**Comment by ntninja** [comment]
Sooo, the detail that is mentioned literally nowhere is that Prosody needs to load the `jitsi_session` module in order for the `room_metadata` module to do anything at all!

So what happened is basically that the `room_metadata` module received the metadata events for the whiteboard (show, hide, encryption key set) but silently discarded them because “they below to no Jitsi meet session”. Because of this other participants in the room were no notified of the change and didn’t see any need to display the whiteboard. The “couldn’t decrypt data” errors then happen when you manually open the whiteboard from another participant and it tries to set its own encryption key (which was again ignored by the metadata module) and then connects to the excalidraw-backend server sending data for the same room but encrypted using a different key – hence the other participant being unable to decrypt it.

TL;DR: Make sure you have **both the `room_metadata` and the `jitsi_session` module enabled in Prosody** to get the whiteboard to work! Also make sure the metadata component is set up and referenced as mentioned elsewhere.

**Comment by damencho** [comment]
This means that room metadata needs: `module:depends("jitsi_session");`
like: https://github.com/jitsi/jitsi-meet/blob/55e75d56fd02940db7ff0bb7c3c86b65312cc0f4/resources/prosody-plugins/mod_muc_lobby_rooms.lua#L26

**Comment by ntninja** [comment]
@damencho 
https://github.com/jitsi/jitsi-meet/pull/14012
Done!
